internal/providers/cloudstack/cloudstack.go

```diff
@@ -70,6 +75,10 @@ func FetchConfig(f resource.Fetcher) (types.Config, report.Report, error) {
 		return fetchConfigFromDevice(f.Logger, ctx, "CONFIG-2")
 	})
 
+	go dispatch("metadata service", func() ([]byte, error) {
+		return fetchConfigFromMetadataService(f)
+	})
+
 	<-ctx.Done()
 	if ctx.Err() == context.DeadlineExceeded {
 		f.Logger.Info("Config drive was not available in time. Continuing without a config...")
```

Comment:
Given we're adding the ability from fetching from metadata the info message should be updated to reflect that.